config/routes.rb

```diff
@@ -112,6 +112,9 @@
 
     get '/vendors' => 'manage_vendors#index'
     post '/vendors' => 'manage_vendors#create'
+
+    get '/providers' => 'providers#index', as: :providers
+    post '/providers' => 'providers#sync'
```

Comment:
at first look `post '/providers'`  suggests that a new provider is created, but actually it is a sync, 
what do you think about changing the route to  '/providers/sync` ?

Reply:
Yes, much better :+1: